.github/workflows/manual.yml

```diff
@@ -0,0 +1,40 @@
+name: actions
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
```

Comment:
Should we also include `nixos/modules/**`? I had a few cases where module option description broke the manual. Of all PRs we have only a few are touching modules.